CONTRIBUTORS.md

```diff
@@ -14,6 +14,7 @@ The following people have contributed to this project to make it possible, and w
 * [Russ Goldin](https://github.com/tagyoureit)
 * [William Lang](https://github.com/williamlang)
 * [Jim Morris](https://github.com/jimbodriven)
+* [BingoDinkus](https://githib.com/BingoDinkus)
```

Comment:
`github` typo

Reply:
One of these days, I'll learn how to type.